app/test_main.py

```diff
@@ -1,3 +1,56 @@
 from app.main import get_human_age
+import pytest
 
-# write your code here
```

Comment:
Make sure you've tests checks all edged situations such as:

if function resive data out of normal range, such as negative numbers, zero, or realy large numbers;
if the function receives an incorrect type of data it raises the correct exception.

Reply:
in the video on the subject of tests, it was clearly stated that writing tests to check the data type is the wrong approach, for this there are type annotations, the error in this case is the responsibility of the developer